modules/ROOT/nav.adoc

```diff
@@ -70,11 +70,15 @@ include::{partialsdir}/attributes.adoc[]
 *** xref:data-sync.adoc#auth[Authentication for {sync-service} Service]
 *** xref:data-sync.adoc#authorization[Authorization for {sync-service} Service]
 
-** xref:device-security.adoc[{device-security-service}]
-*** xref:device-security.adoc#introduction[Introduction]
-*** xref:device-security.adoc#setup[Setting Up]
-*** xref:device-security.adoc#device-trust[Performing Device Trust Checks]
-*** xref:device-security.adoc#cert-pinning[Certificate Pinning]
+** Security
```

Comment:
This should be a link, maybe that's where we explain the diff twixt security service and device security?

Reply:
let me handle this as a separate pr